manim/animation/transform.py

```diff
@@ -144,6 +144,68 @@ def interpolate_submobject(
 
 
 class ReplacementTransform(Transform):
+    """Replace the mobject with target_mobject (like Transform does), and shows the target_mobject (whereas Transform shows the mobject)
+
+    Fades one mobject into another. Transform M1 is shown M2 doies not change, only M1
+
+        Parameters
+        ----------
+        mobject
+            The starting :class:`~.Mobject`.
+        target_mobject
+            The target :class:`~.Mobject`.
+        kwargs
+            Further keyword arguments are passed to the parent class.
+
+        Examples
+        --------
+
+        .. manim:: ReplacementTransformOrTransform
+
+            class ReplacementTransformOrTransform(Scene):
```

Comment:
This example seems like a little excessive just to highlight the difference between the two animations. Plus, writing the latex and the animations would add a significant amount of time needed to build the docs. 

I think it could be be done neatly using squares.

Reply:
I like the text example as users tend to get confused on the usage of this, but maybe we could have 3 text examples and 3 shapes?

Reply:
Some squares with integers in them should achieve the same purpose and be much easier to understand. Another issue with this example is that it's very hard to parse because it's ~40 lines of code

Reply:
Thank you both for your comments! I tried to take into account all of them in the modification. First contribution so it might not be good. Thanks again.